common/changes/office-ui-fabric-react/Implement_OnDismissMenu_2017-12-12-00-47.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Adding onMenuDismissed for ComboBox",
```

Comment:
The format of these messages and usually the PR title : is [ComponentName]: description